extractor/src/main/java/org/schabi/newpipe/extractor/services/media_ccc/extractors/MediaCCCRecentKioskExtractor.java

```diff
@@ -0,0 +1,79 @@
+package org.schabi.newpipe.extractor.services.media_ccc.extractors;
+
+import com.grack.nanojson.JsonObject;
+import org.schabi.newpipe.extractor.exceptions.ParsingException;
+import org.schabi.newpipe.extractor.localization.DateWrapper;
+import org.schabi.newpipe.extractor.stream.StreamInfoItemExtractor;
+import org.schabi.newpipe.extractor.stream.StreamType;
+
+import javax.annotation.Nullable;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class MediaCCCRecentKioskExtractor implements StreamInfoItemExtractor {
+
+    private final JsonObject event;
+
+    public MediaCCCRecentKioskExtractor(final JsonObject event) {
+        this.event = event;
+    }
+
+    @Override
+    public String getName() throws ParsingException {
+        return event.getString("title");
+    }
+
+    @Override
+    public String getUrl() throws ParsingException {
+        return event.getString("frontend_link");
+    }
+
+    @Override
+    public String getThumbnailUrl() throws ParsingException {
+        return event.getString("thumb_url");
+    }
+
+    @Override
+    public StreamType getStreamType() throws ParsingException {
+        return StreamType.VIDEO_STREAM;
+    }
+
+    @Override
+    public boolean isAd() throws ParsingException {
+        return false;
+    }
+
+    @Override
+    public long getDuration() throws ParsingException {
+        return event.getInt("duration");
```

Comment:
do you know what's the difference between length and duration?

Reply:
no. both values are the same in all 100 entries.